### DISCUSSION TITLE
'Adafruit_USBD_MSC' does not name a type

### DISCUSSION THREAD

**Discussion by TigoTas** [adafruit/Adafruit_TinyUSB_Arduino#249] [Q&A]
Hi,

I am really not experienced in these topics, so forgive me if my issue prooves to be idiotic.

I am trying to run the "msc_external_flash_sdcard" out of the box, but the code doesn't compile.

I get a "'Adafruit_USBD_MSC' does not name a type" error message.

Similarly, if I try to run the USB_MSC example from the esp32 library, also as is, I get a "'USBCDC' does not name a type" error message.

My intention is to have the SD card to show as mass storage device on windows, like what LadyAda did on this video.

https://www.youtube.com/watch?v=0bWba0PU4-g

Any help would be appreciated. Perhaps a step-by-step of someone that has had previous experience replicating this functionalty.

Thanks in advance,

[Adafruit_msc_external_flash_sdcard_error.txt](https://github.com/adafruit/Adafruit_TinyUSB_Arduino/files/10663496/Adafruit_msc_external_flash_sdcard_error.txt)

**Comment by hathach** [comment]
which board + cpu you are using ?
